package.json

```diff
@@ -37,8 +37,9 @@
     "docs:dist:gzip": "npm run docs:dist && gulp",
     "start": "npm run build && npm run storybook",
     "native": "(cd ./native && npm run storybook)",
-    "test": "npm run lint && npm run jest && npm run test:native",
-    "test:native": "(cd ./native && npm test)",
+    "check-bpk-dependencies": "node scripts/npm/check-bpk-dependencies.js",
+    "test": "npm run lint && npm run check-bpk-dependencies && npm run jest && npm run test:native",
+    "test:native": "(npm run check-bpk-dependencies && cd ./native && npm test)",
```

Comment:
erm this runs the check twice now?

Reply:
If it only runs it within `./native` it won't check them all, though.